libs/zzz/consts/src/disc.ts

```diff
@@ -158,7 +158,7 @@ export function getDiscMainStatVal(
  */
 export const disc2pEffect: Record<
   DiscSetKey,
-  Partial<Record<StatKey, number>>
+  Partial<Record<PandoStatKey, number>> // TODO: Change to regular StatKey
```

Comment:
Why is it changed to PandoStatKey? old StatKey should still work?

Reply:
StatKey has charLvl, enemyDefIgn, etc, which wont index into `own.char.initial` and such in pando

Reply:
I see, would be nice to include that in the TODO then.